test/e2e/testing_manifests/sno/kernel_parameter_add_rm-child.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: tuned.openshift.io/v1
```

Comment:
all the files under `test/e2e/testing_manifests/sno/` are new and do not seem related to this PR, is this something added from @jmencak's commit ?  or just a result of a non updated rebase?

Reply:
these files are from Jiri's one https://github.com/openshift/cluster-node-tuning-operator/pull/500/files

Not sure if they are necessary